site/blog.html

```diff
@@ -7,11 +7,11 @@
 <div class="post-single-hero bg-color-{{ site.hero.background-color }}">
   <div class="section">
     <div class="section-content">
-      <h1>{{ page.title }}</h1>
+      <h2>{{ page.title }}</h2>
```

Comment:
@amymanion this should be an h1; it's the first thing on the page. if you need it to look differently, add CSS